plugins/toolkit/resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -65,6 +65,7 @@ amazonqFeatureDev.exception.open_diff_failed=Failed to open diff
 amazonqFeatureDev.exception.request_failed=Request failed
 amazonqFeatureDev.exception.retry_request_failed=Retry request failed
 amazonqFeatureDev.exception.uploadCode=Failed to upload code
+amazonqFeatureDev.exception.upload_code=Amazon Q is unable to upload workspace artifacts to S3 for feature development. For more information, see the [Amazon Q documentation](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/security_iam_manage-access-with-policies.html) or contact your network or organization administrator.
```

Comment:
Has UX signed off on this? I'm wondering if it should be "Feature Development" instead of "feature development", since it is the name of the Q capability.

Reply:
UX has signed off on the string, but will reconfirm.

Reply:
Sounds good, thank you